cyclopts/help.py

```diff
@@ -190,16 +190,18 @@ def format_doc(root_app, app: "App", format: str = "restructuredtext"):
 
 
 def _get_choices(type_: Type) -> str:
+    choices: str = ""
     if get_origin(type_) is Union:
         inner_choices = [_get_choices(inner) for inner in get_args(type_)]
         choices = ",".join(x for x in inner_choices if x)
     elif get_origin(type_) is Literal:
         choices = ",".join(str(x) for x in get_args(type_))
     elif isclass(type_) and issubclass(type_, Enum):
         choices = ",".join(x.name.lower().replace("_", "-") for x in type_)
-    else:
-        choices = ""
-
+    elif get_origin(type_) in (list, set, tuple):
+        args = get_args(type_)
```

Comment:
does this work with `Tuple[Literal["foo", "bar"], ...]` (tuple of indeterminate length)?

Reply:
no it dont work ill fix that